mkt/lookup/views.py

```diff
@@ -387,7 +387,7 @@ def _expand_query(q, fields):
 def user_search(request):
     results = []
     q = request.GET.get('q', u'').lower().strip()
-    search_fields = ('username', 'display_name', 'email')
+    search_fields = ('fxa_uid', 'display_name', 'email')
```

Comment:
Actually, I kept it in the template, just made the template use `name` which is a wrapper around `display_name` with a fallback added.